app/main.py

```diff
@@ -1 +1,49 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            food_points = self.appetite
+            print(f"Eating {food_points} food points...")
+            self.is_hungry = False
+            self.appetite = 0
+            return food_points
+        else:
+            return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str,
+                 appetite: int = 3, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite, is_hungry)
+        self.appetite = appetite
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str,
+                 appetite: int = 7, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite, is_hungry)
+        self.appetite = appetite
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list[Animal]) -> int:
+    food_points = sum(animal.appetite
+                      for animal in animals if animal.is_hungry)
+    for animal in animals:
+        animal.feed()
```

Comment:
Function `feed` returns appetite of animal. So, you can rewrite your code to one line
```suggestion
    food_points = sum(animal.feed()
                      for animal in animals if animal.is_hungry)
```